crates/solvers/src/domain/eth/chain.rs

```diff
@@ -0,0 +1,40 @@
+use ethereum_types::U256;
+
+/// A supported Ethereum Chain ID.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum ChainId {
+    Mainnet = 1,
+    Goerli = 5,
+    Gnosis = 100,
```

Comment:
I don't think you're using the integer values anywhere, I don't think you need them.

Reply:
Sure. I liked it because "it was pretty to me" that those are the actual chain ID values, but you are right that they have no place in the domain logic.